PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+Thanks for helping us make a better Snapcraft!
+
+Checklist:
+
+- Have you signed the contributor licence agreement?
+  https://www.ubuntu.com/legal/contributors
+
+- Have you reported a bug for the problem you are fixing?
+  https://bugs.launchpad.net/snapcraft/+filebug
```

Comment:
@kyrofa I'm not quite sure how to do that in a nice way. Please check my push. I'm open to suggestions.